examples/advanced/main.tf

```diff
@@ -111,8 +111,9 @@ module "cos" {
 }
 
 module "cloud_logs_buckets" {
-  source  = "terraform-ibm-modules/cos/ibm//modules/buckets"
-  version = "8.6.2"
+  depends_on = [module.cos]
```

Comment:
why the explicit depends on? There is an implicit dependency one line 132:

> resource_instance_id = module.cos.cos_instance_id

Reply:
Due to the implicit dependency one line 132, once the cos instance is created the buckets in the `cloud_logs_buckets` module and the default bucket in the `cos` module could get created at the same time which causes this error
```
The policy wasn't created because an access policy with identical attributes and roles already exists. Please update the rule in the existing policy (e2e34b45-398c-43b3-a28d-04f7b0d77576), or update the one you're trying to assign to include a different attribute assignment.
```

This is the reason we are skipping the auth policy creation in the `cloud_logs_buckets` module
And added `depends_on` so that only after the complete execution of the `cos` module the `cloud_logs_buckets` module gets executed.